From 9557336c4fc4ac4606ac9e78c239aa689c26e870 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Thu, 26 Sep 2024 17:46:25 +0100
Subject: [PATCH 1288/1350] dts: overlay_map: ramoops-pi4 works on Pi 5

Add the necessary overlay_map entries so that ramoops-pi4 is loaded
with dtoverlay=ramoops on Pi 5.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/overlay_map.dts | 2 ++
 1 file changed, 2 insertions(+)

--- a/arch/arm/boot/dts/overlays/overlay_map.dts
+++ b/arch/arm/boot/dts/overlays/overlay_map.dts
@@ -247,10 +247,12 @@
 	ramoops {
 		bcm2835;
 		bcm2711 = "ramoops-pi4";
+		bcm2712 = "ramoops-pi4";
 	};
 
 	ramoops-pi4 {
 		bcm2711;
+		bcm2712;
 	};
 
 	rpi-cirrus-wm5102 {
